pkg/util/resource/resource.go

```diff
@@ -45,6 +49,12 @@ func GetPodResourceRequests(podSpec *corev1.PodSpec) corev1.ResourceList {
 	reqs := make(corev1.ResourceList)
 
 	for _, container := range podSpec.Containers {
+		gpuLimits, hasGpuLimits := container.Resources.Limits[ResourceGPU]
```

Comment:
Could you add a comment here to explain the rationale?

Reply:
Done.

Reply:
This function is used for more than 1 purposes. Are we confident that this change will not affect the other users of this function?

Reply:
I have checked all references. There will be no other effects.